### DISCUSSION TITLE
Solana (Sol Token ) Transfer

### DISCUSSION THREAD

**Discussion by himmat00007** [WalletConnect/walletconnect-monorepo#4496] [Developer Support] [answered]
can we transfer solana using wallet connect sdk in react-native ?
if yes can i get a code or documentation for this process ?

**Comment by glitch-txs** [comment, marked as answer]
You can use Universal Provider, is a JS library so it works also in React Native.

Example: https://github.com/WalletConnect/web-examples/tree/main/dapps/universal-provider-solana
Docs: https://docs.walletconnect.com/advanced/providers/universal
